example/ndpiReader.c

```diff
@@ -476,10 +484,11 @@ static void help(u_int long_help) {
          "                            | testing purposes in order to compare results across runs\n"
          "  -h                        | This help\n"
          "  -H                        | This help plus some information about supported protocols/risks\n"
-         "  -v <1|2|3>                | Verbose 'unknown protocol' packet print.\n"
+         "  -v <1|2|3|4>              | Verbose 'unknown protocol' packet print.\n"
          "                            | 1 = verbose\n"
          "                            | 2 = very verbose\n"
          "                            | 3 = port stats\n"
+	   "                            | 4 = hash stats\n"
          "  -V <1-4>                  | nDPI logging level\n"
          "                            | 1 - trace, 2 - debug, 3 - full debug\n"
          "                            | >3 - full debug + log enabled for all protocols (i.e. '-u all')\n"
```

Comment:
Either this needs to be changed or your code, because `-v4` has not a different behavior.